docs/mockups/wabachi.html

```
@@ -0,0 +1,242 @@
+                <div class="index-title">Evidence ledger</div>
+                <div class="index-meta">authority → derivation → inference</div>
+              </div>
+              <div id="evidenceLedger"></div>
+            </aside>
+          </div>
+        </div>
+      </main>
+    </div>
+    <div class="backdrop" id="searchBackdrop"></div>
+    <div class="command" id="searchPanel">
+      <input id="searchInput" placeholder="Find symbol, contract, component, finding…" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)